docs/operator-manual/monitor-mode.md

```diff
@@ -1,44 +1,37 @@
 ---
-sidebar_label: "Monitor Mode"
-title: ""
+sidebar_label: "Monitor mode"
+title: "Monitor mode"
+description: The Kubewarden monitor mode and how to activate it.
+keywords: [kubernetes, kubewarden, monitor mode]
 ---
 
-# Monitor mode
+After you deploy a policy on the cluster it starts accepting, rejecting or mutating requests.
 
-When a policy is deployed on the cluster, it will start accepting,
-rejecting or mutating requests. It's a binary operation, either the
-policy is taking decisions on operations it targets, or it's not
-installed at all.
+There is a `monitor` mode so you can deploy a policy to a cluster without it immediately accepting, rejecting or mutating requests.
 
-In order to let users deploy a policy to a cluster, without
-letting it take final decisions, the `monitor` mode has been included.
+The `monitor` mode is a way to deploy policies to the cluster so that:
 
-The `monitor` mode is a way to deploy policies to the cluster in a way
-that:
+1. The policy accepts all requests, as if the policy wasn't installed.
+1. The `policy-server` traces the policy normally.
+Details are included in the trace on whether the request would have been rejected,
+or if a mutation would have been proposed by the policy.
+1. The `policy-server` metrics are updated normally, with the mode included in the metric baggage.
+Therefore, it's easy to filter policies by mode, and focus on the ones deployed via `monitor` mode.
 
-1. All requests that go through the policy will be accepted, as if the
-   policy didn't exist.
-2. The policy result is traced in the `policy-server`
-   normally. Details are included on whether the request would have
-   been rejected, or if a mutation would have been proposed by the
-   policy.
-3. The `policy-server` metrics are updated normally, with the mode
-   included in the metric baggage. Therefore, it's easy to filter policies by
-   mode, and focus on the ones deployed via `monitor` mode.
+The `mode` is an attribute included in the `ClusterAdmissionPolicy` and `AdmissionPolicy` resources.
+There are two values that the `mode` attribute can assume: `monitor` and `protect`.
+The `mode` defaults to `protect` if omitted.
 
-The `mode` is an attribute included in the `ClusterAdmissionPolicy` and `AdmissionPolicy`
-resources. There are two values that the `mode` attribute can assume: `monitor` and `protect`. If the
-`mode` is omitted, it will default to `protect`.
-
-In order to create a policy in monitor mode, all you need to do is to
-include the mode as part of the spec of the resource. For example, in a `ClusterAdmissionPolicy`:
+To create a policy in `monitor mode` you need include the `mode` as part of the specification of the resource.
+For example, as highlighted, in this `ClusterAdmissionPolicy`:
 
 ```yaml
 apiVersion: policies.kubewarden.io/v1alpha2
 kind: ClusterAdmissionPolicy
 metadata:
   name: psp-capabilities
 spec:
+// highlight-next-line
```

Comment:
I don't know how to do that properly. At the end, I always end up adding a comment on the exact line that I want to highlight, for example `mode: monitor # ← `

Reply:
The // highlight-next-line is an instruction to docusaurus to render the next line highlighted. You can see the effect in the preview. (different shades of grey)